server.js

```diff
@@ -1,8 +1,23 @@
 const express = require('express')
 const app = express()
+const mongoose = require('mongoose')
+const URI = 'mongodb+srv://dhruvk:q6GkQB4Kg3jvSHFd@dhruv-bike-info.dga5pzc.mongodb.net/?retryWrites=true&w=majority'
+
+mongoose.connect(URI)
+.then(()=>{
+    app.get('/',(req,res)=>{
+        res.json({Connection_Status:'Connected'})
+    })
+}).catch((err)=>{
```

Comment:
It's better to handle errors by logging them or sending them to an error tracking service, instead of just changing the response of the root endpoint. [medium]